lib/plugins/aws/deployFunction/index.js

```diff
@@ -62,7 +66,8 @@ class AwsDeployFunction {
     return BbPromise.resolve();
   }
 
-  zipFunction() {
+  packageFunction() {
+    this.serverless.cli.log(`Packaging function: ${this.options.function}...`);
```

Comment:
good point @gertjvr, I didn't expect anyone still running without a terminal without unicode 😱

let's go back to 3 dots for terminal output? @flomotlik @ac360